gcp/tests/test_qualtrics_scheduling.py

```
@@ -0,0 +1,352 @@
+
+    def test_connect_no_consent_returns_400(self, connect_payload):
+        connect_payload["CONSENT"] = "No"
+        with (
+            patch.object(
+                _fn1_module, "insert_survey_response", return_value=True
+            ),
+            patch.object(
+                _fn1_module, "publish_connect_scheduling"
+            ) as mock_connect,
+        ):
+            with _make_request(connect_payload):
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)